docs/data/core-blocks.md

```diff
@@ -0,0 +1,86 @@
+# **core/blocks**: Block Types Data
+
+## Selectors 
+
+### getBlockType
+
+Returns a block type by name.
+
+*Parameters*
+
+ * state: Data state.
+ * name: Block type name.
+
+
+### getCategories
+
+Returns all the available categories.
+
+*Parameters*
+
+ * state: Data state.
+
+*Returns*
+
+Categories list.
+
+### getDefaultBlockName
+
+Returns the name of the default block name.
+
+*Parameters*
+
+ * state: Data state.
+
+*Returns*
+
+Default block name.
+
+### getFallbackBlockName
+
+Returns the name of the fallback block name.
+
+*Parameters*
+
+ * state: Data state.
+
+*Returns*
+
+Fallback block name.
+
+## Actions
+
+### addBlockTypes
+
+Returns an action object used in signalling that block types have been added.
+
+*Parameters*
+
+ * blockTypes: Block types received.
+
+
+### removeBlockTypes
+
+Returns an action object used to remove a registered block type.
```

Comment:
I think this would be clearer as `remove a registered block type`. It might also help to say if it will error if the block type isn't found/wasn't previously registered.